content/competitions/_index.html

```diff
@@ -15,9 +15,14 @@ <h1 class="title is-3">UQCS Competitions</h1>
                 We usually run two major competitive events each year: the UQCS CodeJam in Semester 1, and the UQCS Hackathon in Semester 2.
             </p>
 
+            <h4><a href="codejam">UQCS CodeJam 2021</a> (20th March)</h4>
```

Comment:
Can we append the year to these dates?

Similarly, can we append the year to the URL (so it wont have to change in future).